### PR TITLE
Set due date in purchase invoice based on suppler invoice date

### PR DESCRIPTION
**Pull request by rohitwaghchaure**

### CODE REVIEW

File: erpnext/controllers/accounts_controller.py [hunk 1]
```diff
@@ -646,7 +646,7 @@ def group_similar_items(self):
 			self.remove(item)
 
 	def set_payment_schedule(self):
-		posting_date = self.get("posting_date") or self.get("transaction_date")
+		posting_date = self.get("biil_date") or self.get("posting_date") or self.get("transaction_date")
```
Comment (tundebabzy) [inline erpnext/controllers/accounts_controller.py]: Typo. bill_date not biil_date

File: erpnext/controllers/accounts_controller.py [hunk 2]
```diff
@@ -646,7 +646,7 @@ def group_similar_items(self):
...
 		date = self.get("due_date")
 		due_date = date or posting_date
```
Comment (tundebabzy) [inline erpnext/controllers/accounts_controller.py]: `bill_date` is overwritten here except `due_date` is not set.

**Review by tundebabzy — CHANGES_REQUESTED** [review]

**Comment by tundebabzy** [comment]
PR doesn't work correctly:
peek 2018-01-09 13-59

File: erpnext/controllers/accounts_controller.py (hunk 1 above)
Comment (rohitwaghchaure) [reply erpnext/controllers/accounts_controller.py]: Fixed

**Comment by cosimot** [comment]
Hello! Any news on this?

**Pull request merged**